src/Scripting/Core/ObjectFormatter.cs

```diff
@@ -82,70 +95,68 @@ private static DebuggerTypeProxyAttribute GetApplicableDebuggerTypeProxyAttribut
             return null;
         }
 
-        private static bool IsApplicableAttribute(Type type, Type targetType, string targetTypeName)
+        private static bool IsApplicableAttribute(TypeInfo type, TypeInfo targetType, string targetTypeName)
         {
-            return type != null && targetType.IsEquivalentTo(type)
+            return type != null && AreEquivalent(targetType, type)
                 || targetTypeName != null && type.FullName == targetTypeName;
         }
 
+        private static bool AreEquivalent(TypeInfo type, TypeInfo other)
+        {
+            // TODO: does Equals unify NoPIA interfaces?
```

Comment:
(probably doesn't matter that much though...I assume this just means we won't use the DebuggerTypeProxy in some NoPIA scenarios...)